### ISSUE TITLE
Timeline addition to Maps

### ISSUE THREAD

**Issue by doctormo** [farhat-lab/gentb-site#122]
We could use vis.js to add a timeline selector to the maps interface allowing the data to be narrowed by time range and giving the user some understanding of when all the data is.

http://visjs.org/timeline_examples.html

**Comment by mahafarhat** [comment]
Yes, I need to check how much date data we have. You may also to adapt the same visualization to the mutations function we discussed yesterday the zooming in and out and the scrolling are particular attractive.

Maha
